lib/rules/unit-blacklist/README.md

```diff
@@ -94,3 +94,46 @@ a { -moz-border-radius-topright: 40px; }
 ```css
 a { height: 100vmin; }
 ```
+
+### `ignoreMediaFeatureNames: { unit: ["property", "/regex/"] }`
+
+Ignore units for specific feature names.
+
+For example, with `["px", "dpi"]`.
+
+Given:
+
+```js
+{
+  "px": [ "min-width", "/height$/" ],
+  "dpi": [ "resolution" ]  
+}
+```
+
+The following patterns are *not* considered violations:
+
+```css
+@media (min-width: 960px) { }
+```
+
+```css
+@media (max-height: 280px) { }
+```
+
+```css
+@media not (resolution: 300dpi) { }
+```
+
+The following patterns are considered violations:
+
+```css
+a { @media screen and (max-device-width: 500px) { } }
```

Comment:
Can we just have `@media screen and (max-device-width: 500px) {}` here please as I don't believe nested media queries are standard syntax.

Reply:
Done in my last [commit](6a0bc46)